ui/constants/action_types.js

```diff
@@ -467,6 +506,7 @@ export const COMMENT_SUPER_CHAT_LIST_STARTED = 'COMMENT_SUPER_CHAT_LIST_STARTED'
 export const COMMENT_SUPER_CHAT_LIST_COMPLETED = 'COMMENT_SUPER_CHAT_LIST_COMPLETED';
 export const COMMENT_SUPER_CHAT_LIST_FAILED = 'COMMENT_SUPER_CHAT_LIST_FAILED';
 export const COMMENT_FETCH_MY_COMMENTED_CHANNELS_COMPLETE = 'COMMENT_FETCH_MY_COMMENTED_CHANNELS_COMPLETE';
+export const COMMENT_MEMBERS_ONLY_CHAT_TOGGLED = 'MEMBERS_ONLY_CHAT_TOGGLED';
```

Comment:
:small_red_triangle: Please keep the value the same as the name like the rest

Indirectly depending on that pattern to when using redux dev tools ...